web/portal/client/src/entities/Friend/Form.tsx

```diff
@@ -20,10 +20,11 @@ const Form = (props: EntityFormProps): JSX.Element => {
 
   const isInterVpbx = row?.directConnectivity === 'intervpbx';
   const interVpbxEdition = edit && isInterVpbx;
+  const interVpbxNew = !edit && isInterVpbx;
```

Comment:
```suggestion
  const interVpbxNew = create && isInterVpbx;
```
Whould be better but this should work